it/come-dichiaro-riuso.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Dichiarazione di presa a riuso di software
```

Comment:
I would call the file `dichiarazione.md` for consistency with the other files in /riuso